api/filter/http/http_connection_manager.proto

```diff
@@ -12,29 +12,52 @@ import "google/protobuf/duration.proto";
 import "google/protobuf/struct.proto";
 import "google/protobuf/wrappers.proto";
 
+import "validate/validate.proto";
+
+// [#protodoc-title: HTTP connection manager]
+// HTTP connection manager :ref:`configuration overview <config_http_conn_man>`.
+
 message Rds {
   // Configuration source specifier for RDS.
-  ConfigSource config_source = 1;
+  ConfigSource config_source = 1 [(validate.rules).message.required = true];
 
   // The name of the route configuration. This name will be passed to the RDS
   // API. This allows an Envoy configuration with multiple HTTP listeners (and
   // associated HTTP connection manager filters) to use different route
   // configurations.
-  string route_config_name = 2;
+  string route_config_name = 2 [(validate.rules).string.min_len = 1];
 }
 
 message HttpFilter {
   // The name of the filter to instantiate. The name must match a supported
-  // filter.
-  string name = 1;
+  // filter. The built-in filters are:
+  //
+  // * :ref:`envoy.buffer <config_http_filters_buffer>`
```

Comment:
Ideally we would have protodoc.py figure these out, but not worth it for initial docs.

Reply:
It would be kind of a nice if a single line of the comment could be left out as a "comment comment" or something so I could add a TODO for the docs. I can take a look at that.